[2024-04-23T15:17:27.159+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-23T15:17:22.531276+00:00 [queued]>
[2024-04-23T15:17:27.169+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-23T15:17:22.531276+00:00 [queued]>
[2024-04-23T15:17:27.169+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-23T15:17:27.192+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-23 15:17:22.531276+00:00
[2024-04-23T15:17:27.202+0000] {standard_task_runner.py:60} INFO - Started process 4178 to run task
[2024-04-23T15:17:27.206+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-23T15:17:22.531276+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpi90oxojo']
[2024-04-23T15:17:27.208+0000] {standard_task_runner.py:88} INFO - Job 143: Subtask get_files
[2024-04-23T15:17:27.273+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files manual__2024-04-23T15:17:22.531276+00:00 [running]> on host e2cec6840fb2
[2024-04-23T15:17:27.382+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T15:17:22.531276+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T15:17:22.531276+00:00'
[2024-04-23T15:17:27.397+0000] {logging_mixin.py:188} INFO - 3 files found in /opt/***/dags/data
[2024-04-23T15:17:27.401+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_01.json', '/opt/***/dags/data/github_events_03.json', '/opt/***/dags/data/github_events_02.json']
[2024-04-23T15:17:27.478+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240423T151722, start_date=20240423T151727, end_date=20240423T151727
[2024-04-23T15:17:27.556+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T15:17:27.577+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
